net-mail/queue-repair/queue-repair-0.9.0-r2.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit python-single-r1
+
+DESCRIPTION="A toolkit for dealing with the qmail queue directory structure"
+HOMEPAGE="http://pyropus.ca/software/queue-repair/"
+SRC_URI="http://pyropus.ca/software/queue-repair/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}"
+DEPEND="${RDEPEND}"
+
+PATCHES=( "${FILESDIR}/python3.patch" )
```

Comment:
Please include package name and version in the patch, when this was introduced. So `queue-repair-0.9.0-python3.patch`.